WTWSMS/decisions/LI_target_decisions.txt

```diff
@@ -1,4 +1,4 @@
-targetted_decisions = {	
+decisions = {	
```

Comment:
I am not talking about the ban. I am talking about the Forum's error, in which I could not view the Forum or even log my banned account at. I am not ocntesting the ban. I am contesting the poorly implemented, broken system regarding viewing the Forum WHILE banned.